code/datums/elements/transmuted.dm

```diff
@@ -0,0 +1,22 @@
+/**
+ * Marks an atom as having been transmuted and block attempts to retrieve the materials from the atom.
+ */
+/datum/element/transmuted
+	element_flags = ELEMENT_DETACH
+
+/datum/element/transmuted/Attach(datum/target)
+	if(!isatom(target))
+		return ELEMENT_INCOMPATIBLE
+	. = ..()
+	if(. & ELEMENT_INCOMPATIBLE)
+		return
```

Comment:
Are you sure you need this? I don't think anything else does.

Reply:
I really just put it in there for safety.

Reply:
I think it's worth looking at other elements, for example:

```
/datum/element/bsa_blocker/Attach(datum/target)
	if(!isatom(target))
		return ELEMENT_INCOMPATIBLE
	RegisterSignal(target, COMSIG_ATOM_BSA_BEAM, .proc/block_bsa)
	return ..()
```

I'm confused on how your hypothetical could happen.

Reply:
If someone ever changes https://github.com/tgstation/tgstation/blob/de15df7a4753afe2b75a6967052d830d2fc8adf7/code/datums/elements/_element.dm#L20-L26 to return `ELEMENT_INCOMPATIBLE` on a `/atom` for some reason I don't want it to break something.

Reply:
I will absolutely change it to be consistent with the other elements if you insist.

Reply:
The element was removed so...